### ISSUE TITLE
could we train our own language model in our own corpus using Bert-score ?

### ISSUE THREAD

**Issue by gonghouyu** [Tiiiger/bert_score#15]
could we train our own language model in our own corpus using Bert-score ?

**Comment by felixgwu** [comment]
Although we haven't tried it yet, we believe that's possible. 
Please remember to remove the `torch.no_grad()` in the code; otherwise, the gradients won't get through.

**Comment by ShauryaUppal-1Mg** [comment]
In the score() function params bert = '' can I call my own fine tuned bert model version?
